### ISSUE TITLE
MIM 1.0 compliant maken van Geluidgegevenscollectie.herkomstcollectie

### ISSUE THREAD

**Issue by PB-GNM** [Geonovum/IMG#217]
# Aanleiding wijziging
Dit verzoek is aangemeld door Geonovum met helpdesknummer SDIMG-140.
Bij het valideren van IMGeluid tegen het MIM door Geonovum bleek dat het bovenstaande kenmerk het verkeerde stereotype heeft in het model. In principe is het niet zo erg maar hierdoor is het model niet MIM-compliant, en sommige software die MIM modellen verwerkt doet het dan niet meer. 

# Voorgestelde wijziging
Het stereotype van Geluidgegevenscollectie.herkomstcollectie veranderen van Attribuutsoort naar Gegevensgroep zodat het MIM compliant is:
Was:
![image001](https://github.com/Geonovum/IMG/assets/80040145/70052a33-edec-42e0-8b17-f16267d8bf86)
Wordt:
![image002](https://github.com/Geonovum/IMG/assets/80040145/1a22f65f-65aa-43c2-b542-631ca51dd45a)

# Impactanalyse
Geef hier een indicatie van de impact van de wijziging:
- Wie gaat er wat van merken? **softwareleveranciers van software die test op MIM-compliancy**
- Veranderen definities van objecten in de standaard zodanig dat de wijziging impact heeft op de uit te wisselen gegevens? **Nee**
- Is er een herlevering nodig naar de CVGG? **Nee**
- Heeft het impact op het xml-schema? **Nee**
- Is het backward compatible in de zin van validatie op het schema? **Ja**
- Heeft het impact op de regels in IMG? **Nee**
- Heeft het impact op de validatieregels voor de CVGG? **Nee**
- Heeft het impact op de omgevingsregeling? **Nee**
- Is het een X, Y, of Z wijziging? **Z**

# Toelichting
Ruimte voor extra toelichting. Bijvoorbeeld alternatieven die je overwogen hebt.

**Comment by skornsekj** [comment]
@PB-GNM @wilkoquak verandert door deze wijziging de volgorde van de properties, die verwacht wordt?

**Comment by wilkoquak** [comment]
Ik heb als experiment de wijziging doorgevoerd en opnieuw het schema gegenereerd: er kwam exact hetzelfde schema uit. Dus: nee.

**Comment by skornsekj** [comment]
Is dit niet onderdeel van https://github.com/Geonovum/IMG/issues/227

**Comment by wilkoquak** [comment]
Ja. Eigenlijk was het model ook al niet MIM1.1 compliant. Dus deze had ik al eerder als los punt ingebracht.

**Comment by wilkoquak** [comment]
Uitkomst adviesgroep 30 januari: de adviesgroep is akkoord met dit voorstel.
